assets/src/edit-story/components/panels/panel/simplePanel.js

```diff
@@ -28,8 +33,16 @@ import PanelContent from './shared/content';
 
 function SimplePanel({ children, name, title, ...rest }) {
   return (
-    <Panel name={name} {...rest}>
-      <PanelTitle>{title}</PanelTitle>
+    <Panel name={name}>
+      <PanelTitle
+        title={sprintf(
+          /* translators: %s: panel name. */
+          __('%s panel', 'web-stories'),
+          title
+        )}
```

Comment:
Is this necessary? Why are we adding "panel" as a suffix here?

Reply:
Yup, we added this by purpose, because we want the voiceover to speak the text. 

<img width="855" alt="Screen Shot 2020-11-30 at 4 01 53 PM" src="https://user-images.githubusercontent.com/11252886/100679952-708c9500-3325-11eb-9c3c-5ae9672304fb.png">

To confirm, should we keep the 'panel' in the label text?

Reply:
I think either "Size & position" or "Size & position panel" is fine.

Reply:
Keep it short as "Size & position".